### PR TITLE
 [APB-2421][AK,SS] add-code turned into two pages enter-acc-no, enter-utr

### PR DESCRIPTION
**Pull request by arkowalski**

### CODE REVIEW

File: app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala [hunk 1]
```diff
@@ -58,32 +62,55 @@ class MappingController @Inject()(
...
-              case FORBIDDEN =>
-                Ok(
-                  html.add_code(mappingForm.withGlobalError(
-                    "These details don't match our records. Check your account number and tax reference.")))
-              case CONFLICT => Redirect(routes.MappingController.alreadyMapped())
-            }
+        formWithUtr => {
+          request.session.get("mappingArn") match {
+            case Some(arn) =>
+              mappingConnector.createMapping(Utr(formWithUtr.utr.value), Arn(arn)) map { r: Int =>
```
Comment (rvijayan) [inline app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala]: you can remove `r:Int` from here. Please rename `r` to status or response.

Comment (rvijayan) [reply app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala]: The type of `formWithUtr.utr` is Utr, right? In that case you can avoid enclosing that in Utr here.

File: app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala [hunk 2]
```diff
@@ -58,32 +62,55 @@ class MappingController @Inject()(
...
+                r match {
+                  case CREATED   => Redirect(routes.MappingController.complete())
+                  case FORBIDDEN => Redirect(routes.MappingController.noMatch())
+                  case CONFLICT  => Redirect(routes.MappingController.alreadyMapped())
```
Comment (rvijayan) [inline app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala]: Should we worry about any other status here?

File: conf/messages [hunk 3]
```diff
@@ -2,8 +2,21 @@
 servicename.titleSuffix=- Agent services account - GOV.UK
 
 #Connect agent services
-connectAgentServices.title=Connect to your agent services account
-connectAgentServices.label.accountNumber=Agent services account number
+enter-account-number.title=What is your agent services account number?
+enter-account-number.details.heading=For example, XARN-123-4567
```
Comment (rvijayan) [inline conf/messages]: We may not accept the format mentioned in the example ARN. I think we need to remove `-`.

File: conf/messages [hunk 4]
```diff
@@ -2,8 +2,21 @@
...
+enter-account-number.details.summary=Where to find your agent services account number
+enter-account-number.details.content=You can find this on your agent services account homepage. Your account number starts with a letter, then ARN and 7 numbers.
+
+enter-utr.title=What is your business''s Unique Taxpayer Reference(UTR)?
+enter-utr.hint=Enter the last 10 digits only. For example, 12345 67890
```
Comment (rvijayan) [inline conf/messages]: Not sure we have a space in between UTR as well.

File: app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala (hunk 2 above)
Comment (arkowalski) [reply app/uk/gov/hmrc/agentmappingfrontend/controllers/MappingController.scala]: out of scope

**Comment by arkowalski** [comment]
squash to 1 commit

**Pull request merged**